### PR TITLE
TGBOT-173: adds class Author

### PR DESCRIPTION
**Pull request by travka677**

### CODE REVIEW

**Review by max-java — APPROVED** [review]
lgfm

**Review by max-java — APPROVED** [review]
lgfm

**Review by lamaconda — APPROVED** [review]
lgfm

**Review by MarinaSauchuk — APPROVED** [review]
lgfm

File: src/main/java/com/example/demo/model/Author.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
+package com.example.demo.model;
+
+import java.util.Objects;
+
+public class Author {
+    private String name;
+    private int ageOfBirth;
```
Comment (Grideron) [inline src/main/java/com/example/demo/model/Author.java:7]: Rename to "dateOfBirth"

**Review by Grideron — APPROVED** [review]

**Pull request merged**